### ISSUE TITLE
pnpm install fomantic-ui fails

### ISSUE THREAD

**Issue by slavb18** [fomantic/Fomantic-UI#1898]
# Bug Reportm

## Steps to reproduce

1. `pnpm init -y`
2. `pnpm install fomantic-ui`
3. 
## Expected result

fomantic-ui installed without error

## Actual result

```
.../node_modules/fomantic-ui install$ gulp install
│ [14:32:44] Using gulpfile ~/work/examples/test/node_modules/.pnpm/fomantic-ui@2.8.7/node_modules/fomantic-ui/gulpfile.js
│ [14:32:44] Starting 'install'...
│ SUI is a sub-module, skipping end-user install
│ [14:32:44] The following tasks did not complete: install
│ [14:32:44] Did you forget to signal async completion?
└─ Failed in 1.3s
 ERROR  Command failed with exit code 1.
```
## Version

2.8.7

**Comment by lubber-de** [comment]
Please try `pnpx gulp build` instead of `gulp install`..otherwise pnpm as well as yarn is not officially supported and needs somebody to help the project with a proper PR

**Comment by lubber-de** [comment]
Fixed bx #2280
